modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/ScopeContainer.tsx

```diff
@@ -104,52 +104,53 @@ export function ScopeContainer({
 						});
 					}
 
-					setSelectedPanelCategoryValue('');
+					setSelectedPanelCategoryValue(undefined);
 				}}
 				selectedKey={values.scope}
 			/>
-
```

Comment:
Hey @igor-franca, I encountered a bug when trying to change the scope and link category of the panel multiple times using the keyboard, unfortunately this is a common problem with the Picker component, the best way to resolve this is to quickly remove the component from the screen and place it again, forcing the component to rerender. 

So I suggest you create a new state (`loadingPanelLinkSelect` for example) at the top of the component, and in the existing useEffet you can manage the state:


```tsx
	useEffect(() => {
		setPanelCategoryKey(
			values.scope === 'company' ? companies : sites,
			values.panelCategoryKey as string
		);

		setLoadingPanelLinkSelect(true);

		setTimeout(() => setLoadingPanelLinkSelect(false), 100);
		// eslint-disable-next-line react-hooks/exhaustive-deps
	}, [values.scope, companies, sites]);
```

then add the state to make the condition here and place the loading indicator.